4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r

```diff
@@ -0,0 +1,142 @@
+suppressPackageStartupMessages(library(ggplot2))
```

Comment:
nitpick: inline comment for what the library is for e.g.
```suggestion
suppressPackageStartupMessages(library(ggplot2)) # plotting
```

Reply:
Makes sense!